client/state/signup/dependency-store/selectors.js

```diff
@@ -0,0 +1,7 @@
+export function getSignupDependencyStore( state ) {
+	if ( state && state.signup && state.signup.dependencyStore ) {
```

Comment:
Awesome suggestion! Lodash has so many awesome things included inside, that's hard to get to know all of them.